ci-operator/step-registry/mco/conf/day2/add-mcoqe-robot-to-pull-secret/mco-conf-day2-add-mcoqe-robot-to-pull-secret-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+TMP_DIR="/tmp/"
+cluster_pull_secret_file="$TMP_DIR/cluster-pull-secret.json"
+mcoqe_pull_secret_file="$TMP_DIR/mcoqe-pull-secret.json"
+merged_pull_secret_file="$TMP_DIR/merged-pull-secret.json"
+
+
+# Get current pull-secret
+oc get secret pull-secret -n openshift-config '--template={{index .data ".dockerconfigjson" | base64decode}}' > "$cluster_pull_secret_file"
```

Comment:
@sergiordlr I am not sure if every cluster has this secret. if not, better to check if it exists firstly.

Reply:
I have added a check at the beginning of the script. If the secret does not exist, the step is failed.

Reply:
that's ok.

another option: if it does not exist, you could create it for your testing if you would like.  it is low priority option. just FYI.